go-app.js

```diff
@@ -46,6 +48,27 @@ go.utils = {
         return ('' + sum).slice(-1) == check;
     },
 
+    now: function () {
+        return moment();
+    },
```

Comment:
Doing this so I can stub out the time in the tests, they have to match dates that are mostly hardcoded in the fixtures.